### DISCUSSION TITLE
Issue with Bitwarden_RS

### DISCUSSION THREAD

**Discussion by stevewaha** [dani-garcia/vaultwarden#1474] [Q&A]
Hi
It has been working perfectly under docker on CentOS 7 but now I get the following ....
***********************************************************
404: Not Found
The requested resource could not be found.

Rocket
***********************************************************
When trying to get to it via the browser.

From Docker Logs I get
***********************************************************
[INFO] No .env file found.

[WARNING] The following environment variables are being overriden by the config file,
[WARNING] please use the admin panel to make changes to them:
[WARNING] ADMIN_TOKEN

[2021-03-06 11:42:56.900][start][INFO] Rocket has launched from http://0.0.0.0:80
[2021-03-06 11:42:56.907][request][INFO] POST /identity/connect/token
[2021-03-06 11:42:56.907][response][INFO] 404 Not Found
***********************************************************

My docker command is ..

docker run -d --name bitwarden -e ADMIN_TOKEN=XXXX -v /bw-data/:/data/ -p 8080:80 bitwardenrs/server:latest

Internally I do not use a cert .. but I publish Bitwarden via NGINX with cerbot certificate.

Docker thinks it is healthy
***********************************************************
62feae883901   bitwardenrs/server:latest    "/usr/bin/dumb-init …"   21 minutes ago   Up 21 minutes (healthy)   3012/tcp, 0.0.0.0:8080->80/tcp   bitwarden
***********************************************************

Regards

Steve

**Comment by BlackDex** [comment]
Could you check the `/admin/diagnostics` page and see if there are any errors reported there?
And if not, please generate a support string there, and copy/paste it here.

**Comment by stevewaha** [comment]
Hey
The /admin/diagnostics page just has the same 404 Not Found message
Sorry, but how do i generate a support string?
Cheers
